es-design-system/components/ds-es-logo.vue

```diff
@@ -5,7 +5,7 @@
             height: height,
             width: width,
         }"
-        viewBox="0 0 205 45"
+        viewBox="0 0 215 45"
```

Comment:
How did you determine the width should actually be 215px?

Reply:
I didn't. It's a number that came up in conversation when Eric and I were looking over the problem. 210px also seemed to work.